### ISSUE TITLE
Is there an error in the ASAPooling implementation?

### ISSUE THREAD

**Issue by chaolinhuang** [pyg-team/pytorch_geometric#5352]
### 🐛 Describe the bug

When ASAPooling calculates the attention score, there is a linear layer missing in front of softmax. I think the formula in the paper has this layer.

### Environment

* PyG version:
* PyTorch version:
* OS:
* Python version:
* CUDA/cuDNN version:
* How you installed PyTorch and PyG (`conda`, `pip`, source):
* Any other relevant information (*e.g.*, version of `torch-scatter`):

**Comment by rusty1s** [comment]
Thanks for reporting. Isn't this already covered by `self.att`, see [here](https://github.com/pyg-team/pytorch_geometric/blob/master/torch_geometric/nn/pool/asap.py#L102)?
